.cargo/linux_runner_wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+case $1 in
+    *north) sudo -E $@
+    ;;
+    *tests*) sudo -E $@
```

Comment:
Can we limit this to the tests of `north`? The tests of everything else do not need `root`.

Reply:
This is running only `north` and the `tests` (the integration test one) binaries with `sudo`. No other test matches with `*tests*`. I guess, I could try to narrow the match a bit further.